packages/x-components/src/x-modules/search/components/promoted.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <a :href="item.url" class="x-promoted" data-test="promoted">
```

Comment:
Should we emit an event `UserClickedAPromoted` as we do with the results? Then the developer user will be able to use it in a wire (I can't find an specific example to do it know).

Reply:
I'm not sure but this will be useful for the tagging, i dont know how we are going to manage this behaviour.

Reply:
Wont't do